src/components/LeverageSlider/LeverageSlider.scss

```diff
@@ -8,11 +8,14 @@
   .rc-slider-tooltip-inner {
     box-shadow: none;
     padding: 0.465rem 0.8rem;
-    background: var(--color-slate-600);
   }
 
   .rc-slider-tooltip {
     z-index: 5;
+
+    background: rgb(49, 54, 85);
```

Comment:
why not from library, if its one time usage -- ok

Reply:
one time usage